duckduckgo/macos-browser
# Update Attribution logic and debug logs

Task/Issue URL: https://app.asana.com/0/856498667320406/1206957650592842/f
Tech Design URL:
CC:

**Description**:

Update the logic to reduce number of debug signals related to valid use cases.

**Steps to test this PR**:

See the [BSK PR](https://github.com/duckduckgo/BrowserServicesKit/pull/1059) for details.



**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
